ui-components/src/headers/SiteHeader.tsx

```diff
@@ -75,11 +75,11 @@ const SiteHeader = (props: SiteHeaderProps) => {
         >
           <div
             className={`logo__lockup ${getLogoWidthClass()} ${
-              props.logoWidth && "navbar-custom-width"
-            } ${props.imageOnly && "navbar-image-only-container"}`}
+              props.logoWidth ? "navbar-custom-width" : ""
+            } ${props.imageOnly ? "navbar-image-only-container" : ""}`}
```

Comment:
Not sure I understand how this is different than what's already there?

Reply:
for some reason what was there was printing "undefined" instead of nothing.